packages/vscode-lit-plugin/src/test/simple-test.ts

```diff
@@ -15,4 +17,32 @@ suite("Extension Test Suite", () => {
 		const ourId = "runem.lit-plugin";
 		assert.ok(extensionIds.includes(ourId), `Expected ${JSON.stringify(extensionIds)} to include '${ourId}'`);
 	});
+
+	test("We produce a diagnostic", async () => {
+		const config = vscode.workspace.getConfiguration();
+		config.update("lit-plugin.logging", "verbose", true);
+		config.update("lit-plugin.rules.no-missing-element-type-definition", "error", true);
+		const doc = await vscode.workspace.openTextDocument(vscode.Uri.file(path.join(__dirname, "../../src/test/fixtures/missing-elem-type.ts")));
+		await vscode.window.showTextDocument(doc);
+
+		// wait until the TS language server is ready and diagnostics are produced
+		async function getDiagnostics() {
+			for (let i = 0; i < 100; i++) {
+				const diagnostics = vscode.languages.getDiagnostics(doc.uri);
+				if (diagnostics.length > 0) {
+					return diagnostics;
+				}
+				// Is there a better way to wait for the ts server to be ready?
+				// Maybe we can listen for the event that displays and hides the "initializing TS/JS language features" message?
```

Comment:
🙏

Reply:
Eyeballing it, there's still a bit of time between that message going away and the diagnostics being available. Hm.